### ISSUE TITLE
Python 3 / OpenCV

### ISSUE THREAD

**Issue by TrueMophisto** [YenTheFirst/card_scan#15]
Dear YenTheFirst,

Thanks for your amazing code, the video looks awesome. Since I am a total beginner in regards to programming I am currently struggeling to get your code to work with Python3 and the current OpenCV Version.

Did you eventually updated your code? Or would you give a helping hands, which parts need to be updated in order to get it work?

Would be great if you would come back to me.

Regards

**Comment by YenTheFirst** [comment]
The code is currently written against python 2.7. I'll take a look at what incompatibilities there are with python 3.

**Comment by TrueMophisto** [comment]
Did you have a chance to have a look on it?

**Comment by YenTheFirst** [comment]
Not yet. My development machine has been out of service.

**Comment by Fontinalis** [comment]
Hello,
Can I help you to make it compatible with python 3?

**Comment by TrueMophisto** [comment]
Yes, you can! :)

Any references? Where are you located?

Regards,
mophisto

Am 06.10.2016 um 21:33 schrieb Barnabás Pataki <notifications@github.com<mailto:notifications@github.com>>:


Hello,
Can I help you to make it compatible with python 3?

**Comment by Fontinalis** [comment]
I can send you some of my codes in email if you want. I don't want to upload here. And I'm from Hungary. I also sent my proposal on TrueLancer.

**Comment by TrueMophisto** [comment]
Deat Pataki,

are you familiar with OpenCV and did you have already a look into the code?

Can you transfer the given code to Python 3?

For a better understanding I would like to see each section / block of the code with a comment from you to see what the given code is doing.

Also, it is necessary for me that the code will be useable on my personal machine, e.g. please no web based applications.

Can you deliver this kind of code?

Would the price be the same?

Regards,
Ulrich


Am 06.10.2016 um 22:54 schrieb Barnabás Pataki <notifications@github.com<mailto:notifications@github.com>>:


I can send you some of my codes in email if you want. I don't want to upload here. And I'm from Hungary. I also sent my proposal on TrueLancer.

**Comment by Fontinalis** [comment]
Dear Ulrich,

Yes, I am familiar with OpenCV, I took part in a project, that was about automating of the drones' (quadrocopters) controlling by their webcam. This project based on OpenCV in Python, so I have experiance in it. Yes, I already looked into the code. 

Yes, I can transfer it. I created projects in Python 2.7 and also in Python 3. 
Fortunately, I create my codes in blocks with comments, so that won't be problem.

I don't want to do this as a web based application.
Yes, the price would be the same. But here is my email address to contact me (barnabas.vac(at)gmail(dot)com)

Best regards,
Barnabás Pataki
